components/brave_ads/browser/ad_notification_util.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_AD_NOTIFICATION_UTIL_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_AD_NOTIFICATION_UTIL_H_
+
+#include <memory>
+
+namespace message_center {
+class Notification;
+}  // namespace message_center
+
+namespace ads {
+struct AdNotificationInfo;
+}  // namespace ads
+
+namespace brave_ads {
+
+class AdNotification;
+
+const char kAdNotificationUrlPrefix[] = "https://www.brave.com/ads/?";
```

Comment:
prefer `constexpr` to `const char` everywhere

Reply:
Fixed